dist/index.html

```diff
@@ -32,7 +32,7 @@
         </div>
         <div class="row top-description">
             <div class="col-md-12">
-                <p>Dream it. Hack it. Ship it. BarcelonaTech's first collegiate Hackathon. 400 hackers. 36h.</p>
+              <p>Dream it. Hack it. Ship it.<br>BarcelonaTech's first collegiate Hackathon.<br>400 hackers. 36h.</p>
```

Comment:
Indentation is correct after my changes, it was wrong before. I didn't even notice, tbh.